clients/email-setting-client/README.md

```diff
@@ -0,0 +1,43 @@
+# `@epilot/email-settings-client`
+
+API Client for epilot email-settings API.
+
+Uses [`openapi-client-axios`](https://github.com/anttiviljami/openapi-client-axios)
+
+## Getting Started
+
+Install the package:
+
+```bash
+npm install --save-dev @epilot/email-settings-client
+```
+
+Import the package:
+
+```typescript
+import { getClient } from '@epilot/email-settings-client';
+```
+
+Use the client:
+```typescript
+// get typed client
+const client = await getClient();
+
+// call an operation
+const res = await client.testS3();
+```
+
+## BaseURL & Authorization
+
+To pass an authorization header and set up the API url, you can use axios
+defaults:
+
+```typescript
+const client = getClient();
+client.defaults.baseURL = config.API_URL;
+client.defaults.headers['authorization'] = `Bearer ${token}`;
+```
```

Comment:
authentication should be done via `authorizeWithToken`, see the entity client example below.

```
import { authorizeWithToken } from 'epilot-sdk/auth';
import entityClient from 'epilot-sdk/entity-client';

// authorize client using an access token
authorizeWithToken(entityClient, '<my_access_token>');
```